test/dynamo/test_triton_kernels.py

```diff
@@ -893,148 +893,183 @@ def f(x, y):
+                "y_elements": 4,
+            },
+            ["out_ptr"],
+        ],
+        [
+            indirection_kernel,
+            {
+                "in_ptr0": t,
+                "out_ptr": t,
+                "n_elements": 4,
+                "BLOCK_SIZE": 4,
+                "ACTIVATION": "mul2_inplace_kernel",
+            },
+            ["in_ptr0", "out_ptr"],
+        ],
+        [
+            indirection_kernel,
+            {
+                "in_ptr0": t,
+                "out_ptr": t,
+                "n_elements": 4,
+                "BLOCK_SIZE": 4,
+                "ACTIVATION": "add_kernel",
+            },
+            # TODO(oulgen): Multiple functions is not implemented yet
+            ["in_ptr0", "out_ptr"],
+        ],
+        [
+            mul2_inplace_kernel,
+            {"ptr": t, "n_elements": 4, "BLOCK_SIZE": 4},
+            ["ptr"],
+        ],
+        # Cant optimize since the kernel contains a tl.inline_asm_elementwise
+        [
+            inline_asm_kernel,
+            {"X": t, "Y": t, "Z": t, "n": 4, "BLOCK": 4},
+            ["X", "Y", "Z"],
+        ],
+        [
+            add_kernel_with_block_ptr,
+            {
+                "x_ptr": t,
+                "y_ptr": t,
+                "output_ptr": t,
+                "n_elements": 4,
+                "BLOCK_SIZE": 4,
+            },
+            ["output_ptr"],
+        ],
+        [
+            add_kernel_with_import,
+            {
+                "in_ptr0": t,
+                "in_ptr1": t,
+                "out_ptr": t,
+                "n_elements": 4,
+                "BLOCK_SIZE": 4,
+            },
+            ["out_ptr"],
+        ],
+        [
+            atomic_add_kernel,
+            {
+                "in_ptr0": t,
+                "in_ptr1": t,
+                "out_ptr": t,
+                "n_elements": 4,
+                "BLOCK_SIZE": 4,
+            },
+            ["out_ptr"],
+        ],
+        [
+            add_4_times_kernel,
+            {
+                "in_ptr0": t,
+                "in_ptr1": t,
+                "out_ptr": t,
+                "n_elements": 4,
+                "BLOCK_SIZE": 4,
+            },
+            # TODO(oulgen): For loops not implemented yet
+            ["in_ptr0", "in_ptr1", "out_ptr"],
+        ],
+        [
+            cond_op_kernel,
+            {
+                "in_ptr0": t,
+                "in_ptr1": t,
+                "out_ptr": t,
+                "n_elements": 4,
+                "BLOCK_SIZE": 4,
+            },
+            # TODO(oulgen): Dynamic control flow is not implemented yet
+            ["in_ptr0", "in_ptr1", "out_ptr"],
+        ],
+    ]
+    for kernel, inputs, outputs in tests:
+        fn = make_mutation_test(lambda: (kernel, inputs, outputs))
+        name = f"test_mutations_{kernel.fn.__name__}"
+        # Poor way to make test names be unique
+        while name in MutationTests.__dict__:
+            name += "1"
```

Comment:
This will append "1" multiple times to the `name` string. So the test names may end up like `test_name1`, `test_name11`, `test_name111`, etc. Is this intended?

Reply:
Yep, it should be super rare. I only noticed that because i was using indirection kernel twice. Honestly, I dont expect there to be another case. But guarding against it just in case.